apps/opencs/model/tools/pathgridcheck.cpp

```diff
@@ -29,6 +21,12 @@ int CSMTools::PathgridCheckStage::setup()
 
 void CSMTools::PathgridCheckStage::perform (int stage, CSMDoc::Messages& messages)
 {
+    // NOTE: This is horribly inefficient but in order to use signals the entire Stage class
+    // hierarchy needs to be braught under Qt which seems like an overkill for a small
+    // performance gain during verify operations
+    CSMSettings::UserSettings &userSettings = CSMSettings::UserSettings::instance();
```

Comment:
This is problematic, because the verifier runs in a different thread. We can't access the user settings from here. Also, this would allow changing configuration midways through a verifier run.